docs/.vuepress/enhanceApp.js

```diff
@@ -64,6 +64,106 @@ export default ({
+      path: '/ingress-public-port',
+      redirect: "https://kuma.io/docs/1.8.x/reference/kubernetes-annotations/#kuma-io-ingress-public-port"
+    },
+    {
+      name: 'direct-access-services',
+      path: '/direct-access-services',
+      redirect: "https://kuma.io/docs/1.8.x/reference/kubernetes-annotations/#kuma-io-direct-access-services"
+    },
+    {
+      name: 'virtual-probes',
+      path: '/virtual-probes',
+      redirect: "https://kuma.io/docs/1.8.x/reference/kubernetes-annotations/#kuma-io-virtual-probes"
+    },
+    {
+      name: 'virtual-probes-port',
+      path: '/virtual-probes-port',
+      redirect: "https://kuma.io/docs/1.8.x/reference/kubernetes-annotations/#kuma-io-virtual-probes-port"
+    },
+    {
+      name: 'sidecar-env-vars',
+      path: '/sidecar-env-vars',
+      redirect: "https://kuma.io/docs/1.8.x/reference/kubernetes-annotations/#kuma-io-sidecar-env-vars"
+    },
+    {
+      name: 'container-patches',
+      path: '/container-patches',
+      redirect: "https://kuma.io/docs/1.8.x/reference/kubernetes-annotations/#kuma-io-container-patches"
+    },
+    {
+      name: 'builtindns',
+      path: '/builtindns',
+      redirect: "https://kuma.io/docs/1.8.x/reference/kubernetes-annotations/#kuma-io-builtindns"
+    },
+    {
+      name: 'builtinDNSPort',
+      path: '/builtindnsport',
+      redirect: "https://kuma.io/docs/1.8.x/reference/kubernetes-annotations/#kuma-io-builtindnsport"
+    },
+    {
+      name: 'ignore',
+      path: '/ignore',
+      redirect: "https://kuma.io/docs/1.8.x/reference/kubernetes-annotations/#kuma-io-ignore"
+    },
+    {
+      name: 'transparent-proxying-experimental-engine',
+      path: '/transparent-proxying-experimental-engine',
+      redirect: "https://kuma.io/docs/1.8.x/reference/kubernetes-annotations/#kuma-io-transparent-proxying-experimental-engine"
+    },
+    {
+      name: 'envoy-admin-port',
+      path: '/envoy-admin-port',
+      redirect: "https://kuma.io/docs/1.8.x/reference/kubernetes-annotations/#kuma-io-envoy-admin-port"
+    },
+    {
+      name: 'service-account-token-volume',
+      path: '/service-account-token-volume',
+      redirect: "https://kuma.io/docs/1.8.x/reference/kubernetes-annotations/#kuma-io-service-account-token-volume"
+    },
+    {
+      name: 'transparent-proxying-reachable-services',
+      path: '/transparent-proxying-reachable-services',
+      redirect: "https://kuma.io/docs/1.8.x/reference/kubernetes-annotations/#kuma-io-transparent-proxying-reachable-services"
+    },
+    {
+      name: 'transparent-proxying-inbound-v6-port',
+      path: '/transparent-proxying-inbound-v6-port',
+      redirect: "https://kuma.io/docs/1.8.x/reference/kubernetes-annotations/#kuma-io-transparent-proxying-inbound-v6-port"
+    },
+    {
+      name: 'sidecar-drain-time',
+      path: '/sidecar-drain-time',
+      redirect: "https://kuma.io/docs/1.8.x/reference/kubernetes-annotations/#kuma-io-sidecar-drain-time"
+    }
```

Comment:
**Suggestion**: I think the redirection should happen via the Netlify redirect rules that are defined here:

https://github.com/kumahq/kuma-website/blob/3fe10d33405c93d69a2c5b50c0499008a114d16f/docs/.vuepress/config.js#L343-L407

This way the redirect happens before loading the Vuepress application on the client side.

I think the redirects should also dynamically determine what’s the latest stable release so we don’t have to manually update the version string every time we release a new stable version.

Reply:
> I think the redirects should also dynamically determine what’s the latest stable release 

For this **I think** we can use `latest` e.g. https://kuma.io/docs/latest/policies/traffic-log/#add-a-logging-backend redirects to https://kuma.io/docs/1.8.x/policies/traffic-log/#add-a-logging-backend

Reply:
Alright I'll apply changes accordingly

Reply:
This works now but there are some paths that are not following the common redirection syntax like [this one](https://github.com/kumahq/kuma-website/pull/1032/commits/62937958a0408a3e122b1317111005183137a753#:~:text=%60prometheus.metrics.kuma.io/port%20/docs/%24%7Bversions.latestMinor%7D/reference/kubernetes%2Dannotations/%23prometheus%2Dmetrics%2Dkuma%2Dio%2Dport%60%2C) and I can't actually test this because it showing me some privacy-related issue in the browser while testing locally. Is this still right or have some changes in it?

Reply:
@MustkimKhatik I’ve addressed your question in https://github.com/kumahq/kuma-website/pull/1032#discussion_r995438003.